Makefile

```diff
@@ -207,6 +220,57 @@ $(BUILD_DIRECTORY)/electron-$(ELECTRON_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH
 		-s "$(TARGET_PLATFORM)" \
 		-o $@
 
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH)-app: \
+	package.json lib \
+	$(BUILD_DIRECTORY)/node-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/node_modules \
+	| $(BUILD_DIRECTORY)
+	mkdir $@
+	$(foreach prerequisite,$^,$(call execute-command,cp -rf $(prerequisite) $@))
+
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH).js: \
+	$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH)-app \
+	| $(BUILD_DIRECTORY)
+	./scripts/build/concatenate-javascript.sh -e $</lib/cli/etcher.js -o $@
+
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH): \
```

Comment:
Hmmm, so
```
...$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH)...
```
a few lines above, and then
```
...$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH)...
```
here (i.e. the order of `$(APPLICATION_VERSION)` and `$(TARGET_PLATFORM)` has been switched around).
Is that deliberate, or was it a typo? (I suspect the latter)